src/main/resources/assets/dragonsurvival/lang/en_us.json

```diff
@@ -1339,21 +1339,21 @@
   "ds.advancements.transplant_chorus_fruit.title": "Favorite Treat Everywhere",
   "ds.advancements.transplant_chorus_fruit.description": "Transplant a tasty purple flower back home on the Overworld.",
   "ds.advancements.meat_eater.title": "Meat Eater",
-  "ds.advancements.meat_eater.description": "Eat all unique food that is marked as forest dragon food from Dragon Survival. JEI is your friend!",
+  "ds.advancements.meat_eater.description": "Eat all unique forest dragon food from Dragon Survival.",
```

Comment:
At least leave a mention of JEI in at least one place, because it's really important and convenient.